ws/ws-tests.ts

```diff
@@ -55,6 +56,14 @@ var WebSocketServer = WebSocket.Server;
 }
 
 {
+    var httpsserver = https.createServer({});
+    var wss = new WebSocket.Server({ server: httpsserver});
+    var httpserver = http.createServer();
+    var wss = new WebSocket.Server({ server: httpserver});
+}
```

Comment:
Please relax. It's just suggestion. :smile: 
However, Passing Travis CI is more important.